requirements/tests.txt

```diff
@@ -0,0 +1 @@
+# placeholder
```

Comment:
Tests and dev could be the same requirements file for simplicity, no?

Reply:
Honestly, I don't think so. When I stumbled upon this organization structure, at first, it seems complex, but it is the opposite.

Some reasons:

1. This project is kinda small, but if we start to install tools for testing, like plugins and its dependencies, the install time may increase substantially.
2. If I want to use the project in a Docker Container, I want it to be the smallest as possible.
3. As you pointed in your next comment, if we need to auto generate `setuptools`'s requirements, this is not possible for obvious reasons :)
4. Inside `setuptools` there is a directive called:

```python
tests_require = [   ]
```
Which sets the requirements for testing.

I've also checked for other open source projects to see if this approach is kinda a convention, and, for my surprise, it is. Some projects also creates a `docs.in` file that holds tooling for generating docs, like sphinx or swagger.

Anyway, feel free to decide 😄

Reply:
Alright! Looks good then, we can use the current structure. Thanks for explaining!